### PR TITLE
👥 Profiles in Proxy Recorder

### PR DESCRIPTION
**Pull request by Gilad-Gur-Andelman**

### CODE REVIEW

**Comment by Gilad-Gur-Andelman** [comment]
## 🏗 MacOS Build Artifacts are available here👇 

### 1. [🍏 .dmg](https://output.circle-artifacts.com/output/job/c6a1179d-42b6-4a4b-ba44-60601d401c65/artifacts/0/out/make/Loadmill-3.3.29-x64.dmg)

### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/c6a1179d-42b6-4a4b-ba44-60601d401c65/artifacts/0/out/make/zip/darwin/x64/Loadmill-darwin-x64-3.3.29.zip)

**Comment by Gilad-Gur-Andelman** [comment]
## 🏠 Windows build Artifacts are available here👇 

### 1. [🪟 Setup.exe](https://output.circle-artifacts.com/output/job/e8078838-70fc-4080-85a4-191c7adc32a4/artifacts/0/out/make/squirrel.windows/x64/Loadmill-desktop-app-3.3.29-Setup.exe)
### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/e8078838-70fc-4080-85a4-191c7adc32a4/artifacts/0/out/make/zip/win32/x64/Loadmill-win32-x64-3.3.29.zip)

**Comment by Gilad-Gur-Andelman** [comment]
## 🏗 MacOS Build Artifacts are available here👇 

### 1. [🍏 .dmg](https://output.circle-artifacts.com/output/job/cc7574de-b12f-41cd-9541-3bf8e146d2f3/artifacts/0/out/make/Loadmill-3.3.29-x64.dmg)

### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/cc7574de-b12f-41cd-9541-3bf8e146d2f3/artifacts/0/out/make/zip/darwin/x64/Loadmill-darwin-x64-3.3.29.zip)

**Comment by Gilad-Gur-Andelman** [comment]
## 🏠 Windows build Artifacts are available here👇 

### 1. [🪟 Setup.exe](https://output.circle-artifacts.com/output/job/e3ec8f58-a6a2-4402-bbed-fb258820643a/artifacts/0/out/make/squirrel.windows/x64/Loadmill-desktop-app-3.3.29-Setup.exe)
### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/e3ec8f58-a6a2-4402-bbed-fb258820643a/artifacts/0/out/make/zip/win32/x64/Loadmill-win32-x64-3.3.29.zip)

**Comment by Gilad-Gur-Andelman** [comment]
## 🏗 MacOS Build Artifacts are available here👇 

### 1. [🍏 .dmg](https://output.circle-artifacts.com/output/job/ed9639a6-4c61-4320-8dee-f358f225d885/artifacts/0/out/make/Loadmill-3.3.30-x64.dmg)

### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/ed9639a6-4c61-4320-8dee-f358f225d885/artifacts/0/out/make/zip/darwin/x64/Loadmill-darwin-x64-3.3.30.zip)

**Comment by Gilad-Gur-Andelman** [comment]
## 🏗 MacOS Build Artifacts are available here👇 

### 1. [🍏 .dmg](https://output.circle-artifacts.com/output/job/47a472bd-cefa-4930-98c9-1fe46a9c9a8f/artifacts/0/out/make/Loadmill-3.3.30-x64.dmg)

### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/47a472bd-cefa-4930-98c9-1fe46a9c9a8f/artifacts/0/out/make/zip/darwin/x64/Loadmill-darwin-x64-3.3.30.zip)

**Comment by Gilad-Gur-Andelman** [comment]
## 🏠 Windows build Artifacts are available here👇 

### 1. [🪟 Setup.exe](https://output.circle-artifacts.com/output/job/d9ab9093-7a85-4dc2-8d43-fd0f66b8a104/artifacts/0/out/make/squirrel.windows/x64/Loadmill-desktop-app-3.3.30-Setup.exe)
### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/d9ab9093-7a85-4dc2-8d43-fd0f66b8a104/artifacts/0/out/make/zip/win32/x64/Loadmill-win32-x64-3.3.30.zip)

File: src/renderer-process/proxy-view/ui/proxy-dashboard-header.tsx [hunk 1]
```diff
@@ -16,18 +15,23 @@ export const ProxyDashboardHeader = ({
...
   setFilterRegex,
   setIsDownloadInProgress,
+  onSelectProfile,
   setSelectedSuite,
   suites,
-  isFetchingSuites,
-  onSearchSuites,
 }: ProxyDashboardHeaderProps): JSX.Element => (
   <div
     className='proxy-dashboard-header'
```
Comment (onniaa) [inline src/renderer-process/proxy-view/ui/proxy-dashboard-header.tsx:37]: I would suggest changing `proxy-dashboard-header` from-
```
align-items: center;
display: flex;
justify-content: space-between;
  ```
  to-
  ```
  align-items: center;
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  ```
  And add to `ConnectionInfoBanner` `Paper` style-
  ```
  justify-self: center;
  ```
  
  I think it looks a bit better when working w/o a monitor.

**Review by onniaa — APPROVED** [review]

**Comment by Gilad-Gur-Andelman** [comment]
## 🏗 MacOS Build Artifacts are available here👇 

### 1. [🍏 .dmg](https://output.circle-artifacts.com/output/job/91630dba-eaca-47dd-9a5e-909fb23c75c7/artifacts/0/out/make/Loadmill-3.3.30-x64.dmg)

### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/91630dba-eaca-47dd-9a5e-909fb23c75c7/artifacts/0/out/make/zip/darwin/x64/Loadmill-darwin-x64-3.3.30.zip)

**Comment by Gilad-Gur-Andelman** [comment]
## 🏠 Windows build Artifacts are available here👇 

### 1. [🪟 Setup.exe](https://output.circle-artifacts.com/output/job/1fb1ffb3-8deb-49ea-845e-398eef20f6aa/artifacts/0/out/make/squirrel.windows/x64/Loadmill-desktop-app-3.3.30-Setup.exe)
### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/1fb1ffb3-8deb-49ea-845e-398eef20f6aa/artifacts/0/out/make/zip/win32/x64/Loadmill-win32-x64-3.3.30.zip)

**Pull request merged**